drivers/wifi/siwx91x/siwx91x_wifi.c

```diff
@@ -164,7 +164,7 @@ static sl_status_t siwx91x_wifi_module_stats_event_handler(sl_wifi_event_t event
 	case STATE_UNASSOCIATED:
 		wifi_mgmt_raise_disconnect_result_event(sidev->iface,
 							WIFI_REASON_DISCONN_SUCCESS);
-
+		sidev->ps_params.enabled = WIFI_PS_DISABLED;
```

Comment:
This change looks weird. `sidev->ps_params` store the PS parameters requested by the user. Only `siwx91x_set_power_save()` is supposed to write it.

Reply:
This is not the Zephyr API.